include/robot_interfaces/robot_data.hpp

```diff
@@ -55,7 +55,7 @@ class RobotData
 
     RobotData(size_t history_length = 1000,
               bool use_shared_memory = false,
-              std::string shared_memory_address = "")
+              __attribute__((unused)) std::string shared_memory_address = "")
```

Comment:
i don't understand this, could you explain this?

Reply:
It suppresses the "unused variable" warning for this argument and makes it explicit in the code that we are aware that this is currently unused but still want to keep it for now.
I will add a comment in the code